src/ripple/protocol/STObject.h

```diff
@@ -401,7 +401,7 @@ class STObject
         if (!cf)
             throw std::runtime_error ("Wrong field type");
 
-        cf->setValue (value);
+        cf->setValue (std::move (value));
```

Comment:
Thanks gents.  I'm reading up now.  It looks like the `static_assert` is more like adding armor plating than belt and suspenders.  Sorry for the noise.